torch/_dynamo/variables/builder.py

```diff
@@ -1848,7 +1848,7 @@ def __call__(self, tx, value) -> VariableTracker:
             return SourcelessBuilder.wrap_constant_literal(value)
```

Comment:
this is the code path that I prefer device and dtype to take

Reply:
https://github.com/pytorch/pytorch/pull/112284

Reply:
@drisspg since this PR includes a test, could we close that in favour of this one?

Reply:
Forsure, tbh I wasn't able to find any sourceless builder tests and was curious how where to add one, will close